### ISSUE TITLE
Blur problem with Typeahead.js when using Bootstrap 3

### ISSUE THREAD

**Issue by chris-peng-1244** [bootstrap-tagsinput/bootstrap-tagsinput#253]
![screenshot 2014-11-04 17 19 49](https://cloud.githubusercontent.com/assets/4301647/4897195/ce5c2008-6403-11e4-9f8a-66419e91a5bd.png)
I typed "car" and there is no hint of it,  then I press "enter", the "car" label was successfully added, but when I unfocus my cursor from the input, something happened.
![screenshot 2014-11-04 17 19 58](https://cloud.githubusercontent.com/assets/4301647/4897194/cd8fe8ee-6403-11e4-8f71-76de6c83c4f2.png)
I checked the source code of typeahead.bundle.js, and find this

``` js
1001  _onBlur: function onBlur() {
                this.resetInputValue();
                this.trigger("blurred");
            },
```

it resets the value of the input every time I blured from the input, even if tagsinput had already cleaned its value.
I kept on checking the source code of tagsinput, then I found this

``` js
320  self.$input.typeahead(null, typeaheadjs).on('typeahead:selected', $.proxy(function (obj, datum) {
            if (typeaheadjs.valueKey)
              self.add(datum[typeaheadjs.valueKey]);
            else
              self.add(datum);
            self.$input.typeahead('val', '');
          }, self));
```

I thought maybe I could write something like:

``` js
self.$input.typeahead(null, typeaheadjs).on('typeahead:selected typeahead:blur'
```

That didn't work. So I just comment out  `this.resetInputValue();`, and it worked!
I kinda hate this solution, and wish you could give me some insights while I keep learning the source code of both tagsinput and typeahead. Thanks!

**Comment by relaxnow** [comment]
I'm having the same problem but the fix for #236 causes my form to submit despite errors, so I'm forced to hack the lib for now :disappointed: .

**Comment by zspitzer** [comment]
what about binding to the typeahead:change event?
